warp/src/raygun/mod.rs

```diff
@@ -103,9 +104,9 @@ pub enum MessageEventKind {
         did_key: DID,
         event: MessageEvent,
     },
-    ConversationSettingsUpdated {
+    ConversationPermissionsUpdated {
         conversation_id: Uuid,
-        settings: ConversationSettings,
+        permissions: HashMap<DID, Vec<GroupPermission>>,
```

Comment:
I think it might be best to split this to indicate the identity and what permission was added/removed (maybe a `kind` field). Thoughts?

Reply:
done